### PR TITLE
Use LESS for CSS

### PR DESCRIPTION
**Pull request by McGiverGim**
This is a replacement of CSS by LESS: https://lesscss.org/

LESS is basically CSS but with some things more that you can use or not:
- Structured rules
- Imports
- Variables
- ...

You can see some of them here: https://lesscss.org/features/

This is only a proof of concept. If we think is a good idea, I will move to less all the css. I will do it using automatic tools, so we will don't use some of the advanced functionalities, but it opens the possibility to use it in a future.

### CODE REVIEW

**Comment by chmelevskij** [comment]
Some reasons I usually don't use less/sass nowadays:
1. CSS has come a long way, so there is not the much use for it for most basic features like variables, layout helpers and colour functions are in the core now.
2. We are moving towards Vue, so it provides as way to encapsulate styles closer to the components so importing is not that much needed.
3. Even if we do end up needing some fancy features, I find https://postcss.org a better alternative since you can pull in only what you need, like if you need loops, some mixins or similar, so a bit quicker builds as you only use subset of syntax.

I think moving towards components in Vue and then switching to have more encapsulation, like enabling hashed class names etc. is a better way forward.

But if we feel that this would help us in the meantime, don't see why not. 😊 If it comes to it we can add less loader to Vue parts as well.

**Comment by McGiverGim** [comment]
Ok, I think I will continue with this then. Maybe in the future (a long future) is not needed when all is in the Vue components, but then we only need to take the CSS generated and replace the less files.

Starting from scratch this can help a lot, now that we only make little changes it will be less used, but can help with some things.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Review by chmelevskij — DISMISSED** [review]

**Comment by McGiverGim** [comment]
When we want. I think is better to wait, I hope the changes until the release will be minimum...

**Comment by limonspb** [comment]
Conflicts, @McGiverGim :)

**Comment by McGiverGim** [comment]
Rebased, please merge this ASAP, any change in any CSS will break it again...

**Review by chmelevskij — APPROVED** [review]

File: src/css/tabs/receiver.less [hunk 1]
```diff
@@ -0,0 +1,707 @@
...
+				&:last-child {
+					border-top-right-radius: 3px;
+					border-right: 0;
+				}
+			}
+			td {
+				text-align: left;
+				padding: 4px 4px 4px 6px;
+				border-right: 1px solid var(--subtleAccent);
+				padding: 1px;
```
Comment (haslinghuis) [inline src/css/tabs/receiver.less:188]: duplicate "padding"

**Comment by McGiverGim** [comment]
Fixes as per review...

**Review by haslinghuis — APPROVED** [review]

**Review by limonspb — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Comment by Asizon** [comment]
Hello @McGiverGim, seems that there is an issue with Less on Cordova:

Screenshot_20220709_130021_com betaflight betaflight_configurator_debug

**Comment by McGiverGim** [comment]
Open an issue to not forgot, I will be out two weeks 😉

**Pull request merged**